ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/NamespaceView/NamespaceViewPage.tsx

```diff
@@ -37,7 +161,149 @@ function NamespaceViewPage() {
+                                onPerPageSelect={(_, newPerPage) => setPerPage(newPerPage)}
+                                isCompact
+                            />
+                        </ToolbarItem>
+                        <ToolbarGroup aria-label="applied search filters" className="pf-u-w-100">
+                            <SearchFilterChips
+                                onFilterChange={onFilterChange}
+                                filterChipGroupDescriptors={searchOptions.map(
+                                    ({ label, value }) => {
+                                        return {
+                                            displayName: label,
+                                            searchFilterName: value,
+                                        };
+                                    }
+                                )}
+                            />
+                        </ToolbarGroup>
+                    </ToolbarContent>
+                </Toolbar>
+                <TableComposable borders={false}>
+                    <Thead noWrap>
+                        <Tr>
+                            <Th sort={getSortParams('Namespace')} width={30}>
+                                Namespace
+                            </Th>
+                            <Th sort={getSortParams('Namespace Risk Priority')}>Risk priority</Th>
+                            <Th sort={getSortParams('Cluster')}>Cluster</Th>
+                            <Th sort={getSortParams('Deployment Count')}>Deployments</Th>
+                            <Th>Labels</Th>
+                        </Tr>
+                    </Thead>
+                    <Tbody>
+                        {tableUIState.type === 'ERROR' && (
+                            <Tr>
+                                <Td colSpan={5}>
+                                    <TableErrorComponent
+                                        error={tableUIState.error}
+                                        message="An error occurred. Try refreshing again"
+                                    />
+                                </Td>
+                            </Tr>
+                        )}
+                        {tableUIState.type === 'LOADING' && (
+                            <Tr>
+                                <Td colSpan={5}>
+                                    <Bullseye>
+                                        <Spinner isSVG aria-label="Loading table data" />
+                                    </Bullseye>
+                                </Td>
+                            </Tr>
+                        )}
+                        {tableUIState.type === 'EMPTY' && (
+                            <Tr>
+                                <Td colSpan={5}>
+                                    <Bullseye>
+                                        <EmptyStateTemplate
+                                            title="No pending exception requests"
```

Comment:
```suggestion
                                            title="There are currently no namespaces"
```

Reply:
Ah darn title. I caught it during the copy-paste in the actual text. I forgot about the title 🤦🏼